addon/ShareProvider.js

```diff
@@ -403,7 +404,14 @@ ShareProvider.prototype = {
         if (!node || node.id !== id) {
           return;
         }
-        node.setAttribute("image", data.url("content/img/glyph-share-16.svg"));
```

Comment:
Does anything still use the generic "glyph-share-16.svg", or should we delete that now that we're moving towards OS-specific iconography?